go/libraries/doltcore/sqle/enginetest/dolt_queries_merge.go

```diff
@@ -4517,6 +4517,34 @@ var ThreeWayMergeWithSchemaChangeTestScripts = []MergeScriptTest{
 			},
 		},
 	},
+	{
+		// Repro for issue in: https://github.com/dolthub/dolt/pull/6496
+		Name: "drop a column, schema contains BLOB/AddrEnc columns",
+		AncSetUpScript: []string{
+			"set autocommit = 0;",
+			"CREATE table t (pk int primary key, col1 int, col2 text, col3 varchar(10));",
+			"INSERT into t values (1, 10, 'a', 'b'), (2, 20, 'c', 'd');",
+		},
+		RightSetUpScript: []string{
+			"INSERT into t values (300, 30, 'e', 'f');",
+		},
+		LeftSetUpScript: []string{
```

Comment:
Maybe I'm confused, but wasn't the bug triggered by a drop on the right-side of the merge?

Reply:
good eye! the test runner runs each schema merge test in both directions. it's been pretty helpful in catching several bugs. you're right that this bug only shows up in one direction.